browser/brave_browser_features.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_browser_features.h"
+
+namespace features {
+
+// Cleanup Session Cookies on browser restart if Session Restore is enabled.
+BASE_FEATURE(kBraveCleanupSessionCookiesOnSessionRestore,
+             "BraveCleanupSessionCookiesOnSessionRestore",
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
do we need to expose that feature on Android if we never do a clean up?

Reply:
With this feature enabled on Android we will do a cleanup on browser restart, but will never do a restore, because there's no "browser was restarted" event available.